lms/djangoapps/discussion/rest_api/api.py

```diff
@@ -879,6 +878,21 @@ def _handle_read_field(api_content, form_value, user, cc_content):
         api_content["unread_comment_count"] = 0
 
 
+def _handle_pinned_field(pin_thread: bool, cc_content: Thread, user: User):
+    """
+    Pins or unpins a thread
+
+    Args:
```

Comment:
I'd follow the docstring convention how it's used elsewhere and rename it `Arguments:`